### PR TITLE
Added GePpeTto card

### LINKED ISSUE #4089: GePpeTto!
# 🌟 New model addition

## Model description

GePpeTto is a GPT-2 model for italian. For further details checkout the paper: https://arxiv.org/abs/2004.14253

## Open source status

* [X ] the model implementation is available: we used huggingface's GPT-2 implementation.
* [X ] the model weights are available: you can find the model here: https://github.com/LoreDema/GePpeTto
* [X ] who are the authors: @LoreDema @michelecafagna26 @malvinanissim @Falice1977 @marcoguerini

### PR DESCRIPTION
**Pull request by LoreDema**

### CODE REVIEW

File: model_cards/LorenzoDeMattei/GePpeTto.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+---
+language: Italian
```
Comment (julien-c) [inline model_cards/LorenzoDeMattei/GePpeTto.md:2]:
```suggestion
language: italian
```

File: model_cards/LorenzoDeMattei/GePpeTto.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+| Domain | Perplexity |
+|---|---|
+| Wikipedia | 26.1052 |
+| ItWac | 30.3965 |
+| Legal | 37.2197 |
+| News | 45.3859 |
+| Social Media | 84.6408 |
+
+For furhter details, qualitative analysis and human evaluation check out: https://arxiv.org/abs/2004.14253
```
Comment (julien-c) [inline model_cards/LorenzoDeMattei/GePpeTto.md:44]:
```suggestion
For further details, qualitative analysis and human evaluation check out: https://arxiv.org/abs/2004.14253
```

File: model_cards/LorenzoDeMattei/GePpeTto.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+```
+
+## Example using GPT2LMHeadModel
+
+```python
+from transformers import GPT2Tokenizer, GPT2LMHeadModel
+
+tokenizer = GPT2Tokenizer.from_pretrained('LorenzoDeMattei/GePpeTto')
+model = GPT2LMHeadModel.from_pretrained(
+    'LorenzoDeMattei/GePpeTto', pad_token_id = tokenizer.eos_token_id
```
Comment (julien-c) [inline model_cards/LorenzoDeMattei/GePpeTto.md:66]: If the pad_token_id is in your model's config.json you shouldn't need this? (cc @patrickvonplaten)

File: model_cards/LorenzoDeMattei/GePpeTto.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+)
+
+print('Output:\n' + 100 * '-')
+for i, sample_output in enumerate(sample_outputs):
+    print(
+        '{}: {}'.format(
+            i, tokenizer.decode(sample_output, skip_special_tokens = True)
+        )
+    )
+```
```
Comment (julien-c) [inline model_cards/LorenzoDeMattei/GePpeTto.md:88]: You can also use our new `TextGenerationPipeline` to do this, cc @patrickvonplaten

File: model_cards/LorenzoDeMattei/GePpeTto.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+
+Output is,
+
+```text
+Output:
+----------------------------------------------------------------------------------------------------
+0: Wikipedia Geppetto
+
+Geppetto è una città degli Stati Uniti d'America, situata nello Stato dell'Iowa, nella Contea di Greene.
```
Comment (julien-c) [inline model_cards/LorenzoDeMattei/GePpeTto.md:97]: Great generation here =)

**Review by julien-c — APPROVED** [review]
That's great!

File: model_cards/LorenzoDeMattei/GePpeTto.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+
+Training parameters:
+
+- GPT-2 small configuration
+- vocabulary size: 30k
+- Batch size: 32
+- Block size: 100
+- Adam Optimizer
+- Initial learning rate: 5e-5
+- Warm up steps: 10k
```
Comment (julien-c) [inline model_cards/LorenzoDeMattei/GePpeTto.md:32]: Do you have a link/screenshot to a Tensorboard or to another experiment tracker for the training?

File: model_cards/LorenzoDeMattei/GePpeTto.md (hunk 3 above)
Comment (patrickvonplaten) [reply model_cards/LorenzoDeMattei/GePpeTto.md:66]: Also think it would be a bit cleaner to explicitly set the `pad_token_id` in the config.

**Comment by patrickvonplaten** [comment]
Awesome :-)

File: model_cards/LorenzoDeMattei/GePpeTto.md (hunk 3 above)
Comment (LoreDema) [reply model_cards/LorenzoDeMattei/GePpeTto.md:66]: right, I will fix it

File: model_cards/LorenzoDeMattei/GePpeTto.md (hunk 4 above)
Comment (LoreDema) [reply model_cards/LorenzoDeMattei/GePpeTto.md:88]: Makes sense, but I see in the latest package version (2.8.0) available on pypi TextGenerationPipeline is not there yet. Might be safer to wait until it is included in the latest release?

File: model_cards/LorenzoDeMattei/GePpeTto.md (hunk 5 above)
Comment (LoreDema) [reply model_cards/LorenzoDeMattei/GePpeTto.md:97]: =)

File: model_cards/LorenzoDeMattei/GePpeTto.md (hunk 4 above)
Comment (julien-c) [reply model_cards/LorenzoDeMattei/GePpeTto.md:88]: Yes you're right

File: model_cards/LorenzoDeMattei/GePpeTto.md (hunk 6 above)
Comment (LoreDema) [reply model_cards/LorenzoDeMattei/GePpeTto.md:32]: sorry we just logged the loss in a file, we didn't use any visual tracker

**Pull request merged**